invenio_records_rest/facets.py

```diff
@@ -138,11 +138,37 @@ def _query_filter(search, urlkwargs, definitions):
 
     return (search, urlkwargs)
 
+def without_nested_subtypes(facet_filters, facet_names):
+    """Remove the nested subtypes from the filter.
 
-def _aggregations(search, definitions):
+    Example: If `CMS` from Experiment type is selected
```

Comment:
Very minor: we tend to use examples (or maybe it is only me 🤣) not domain specific, so that it is more clear. In RSK, we are using resource types [here](https://github.com/inveniosoftware/react-searchkit/blob/master/src/lib/state/selectors/query.test.js#L17).